pkgs/desktops/deepin/dde-network-utils/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchFromGitHub, qmake, pkgconfig, qttools,
+  dde-qt-dbus-factory, proxychains, which, deepin }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "dde-network-utils";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = version;
+    sha256 = "1m6njld06yphppyyhygz8mv4gvq2zw0676pbls9m3fs7b3dl56sv";
+  };
+
+  nativeBuildInputs = [
+    qmake
+    pkgconfig
+    qttools
+    deepin.setupHook
+  ];
+
+  buildInputs = [
+    dde-qt-dbus-factory
+    proxychains
+    which
+  ];
+
+  postPatch = ''
+    searchHardCodedPaths  # for debugging
+    patchShebangs translate_generation.sh
+    fixPath ${proxychains} /usr/bin/proxychains4 networkworker.cpp
+  '';
+
+  postFixup = ''
+    searchHardCodedPaths $out  # for debugging
```

Comment:
`searchHardCodedPaths` is a shell function that looks for occurrences of FHS hard coded paths in the source code. It helps a lot in fixing this kind of issue. I am keeping it in the derivation to easily spot them.

It is boring to add it while testing, and then remove it when everything works, and repeat this at every update. Because of that I left it in the derivation.

Is that ok?

Reply:
Is that ok?

Reply:
Sounds good to me. Probably would be something you'd want to check on each on update.